### PR TITLE
Update privacy-policy.md

### PR DESCRIPTION
**Pull request by Aajaantaa**
I suggested change to the text under "International transfers of your personal data" as the law has been updated. Also, replaced contact info of Jonas with mine.

### CODE REVIEW

**Review by odinho — APPROVED** [review]

File: privacy-policy.md [hunk 1]
```diff
@@ -370,8 +371,8 @@ authority.
 
 ## Data protection officer
 
-Our data protection officer's contact details are: Jonas Lybech Jensen,
-privacy@whereby.com
+Our data protection officer's contact details are: Ajanta Hossain Moushumi,
+DPO@whereby.com
```
[the remark this answers is not in the document]
Comment (Aajaantaa) [reply privacy-policy.md:375]: you are right!

File: privacy-policy.md [hunk 2]
```diff
@@ -271,11 +271,12 @@ Norway, United Kingdom and United States. The hosting facilities for Account
 information stored by Whereby are situated in Ireland. The hosting
 facilities for Usage information are situated in Ireland and the
 United States. Transfers to the United States will be protected by
-appropriate safeguards, namely the use of standard data protection
-clauses adopted or approved by the European Commission, a copy of
+appropriate safeguards, namely the use of Standard Contractual Clause (SCC) adopted or approved by the European Commission, a copy of
 which can be obtained from
 https://ec.europa.eu/info/law/law-topic/data-protection/international-dimension-data-protection/standard-contractual-clauses-scc_en. 
 
+Please note that, for new data transfers, we will continue to rely on the old SCCs until September 27,2021. For existing transfer, we will continue to rely on SCCs until December 27, 2022, by which time all data transfers relying on the old SCCs will be moved over to the new SCCs adopted and approved by the European Commission, a copy of which can be obtained from https://ec.europa.eu/info/law/law-topic/data-protection/international-dimension-data-protection/standard-contractual-clauses-scc/standard-contractual-clauses-international-transfers_en
```
[the remark this answers is not in the document]
Comment (Aajaantaa) [reply privacy-policy.md:278]: Yes, absolutely! We should do the same with the other link that was already, one paragraph above.

[the remark this answers is not in the document]
Comment (Aajaantaa) [reply privacy-policy.md:278]: this one:   namely the use of the Standard Contractual Clause (SCC), hyperlinking the text "Standard Contractual Clause (SCC)" with the link following

File: privacy-policy.md [hunk 3]
```diff
@@ -271,11 +271,17 @@ Norway, United Kingdom and United States. The hosting facilities for Account
 information stored by Whereby are situated in Ireland. The hosting
 facilities for Usage information are situated in Ireland and the
 United States. Transfers to the United States will be protected by
-appropriate safeguards, namely the use of standard data protection
-clauses adopted or approved by the European Commission, a copy of
+appropriate safeguards, namely the use of Standard Contractual Clause
+ (SCC) adopted or approved by the European Commission, a copy of
 which can be obtained from
 https://ec.europa.eu/info/law/law-topic/data-protection/international-dimension-data-protection/standard-contractual-clauses-scc_en.
```
Comment (odinho) [inline privacy-policy.md:277]:
```suggestion
appropriate safeguards, namely the use of [Standard Contractual Clause (SCC)](
https://ec.europa.eu/info/law/law-topic/data-protection/international-dimension-data-protection/standard-contractual-clauses-scc_en)
adopted or approved by the European Commission. 
```

File: privacy-policy.md [hunk 4]
```diff
@@ -271,11 +271,17 @@ Norway, United Kingdom and United States. The hosting facilities for Account
...
+appropriate safeguards, namely the use of Standard Contractual Clause
+ (SCC) adopted or approved by the European Commission, a copy of
 which can be obtained from
 https://ec.europa.eu/info/law/law-topic/data-protection/international-dimension-data-protection/standard-contractual-clauses-scc_en. 
 
+Please note that, for new data transfers, we will continue to rely on
+the old SCCs until September 27,2021. For existing transfer, we will
+continue to rely on SCCs until December 27, 2022, by which time all data
+transfers relying on the old SCCs will be moved over to the [new SCCs
+adopted and approved by the European Commission](https://ec.europa.eu/info/law/law-topic/data-protection/international-dimension-data-protection/standard-contractual-clauses-scc/standard-contractual-clauses-international-transfers_en)
```
Comment (odinho) [inline privacy-policy.md:282]:
```suggestion
adopted and approved by the European Commission](
https://ec.europa.eu/info/law/law-topic/data-protection/international-dimension-data-protection/standard-contractual-clauses-scc/standard-contractual-clauses-international-transfers_en)
```

**Pull request merged**